The following modules were not unloaded:
  (Use "module --force purge" to unload all):

  1) XALT/minimal   2) slurm   3) NeSI
Lmod has detected the following error: The following module(s) are unknown:
"gcc/9.3.0"

Please check the spelling or version number. Also try "module spider ..."
It is also possible your cache file is out-of-date; it may help to try:
  $ module --ignore_cache load "gcc/9.3.0"

Also make sure that all modulefiles written in TCL start with the string
#%Module



Mon Oct 21 10:25:34 2024       
+---------------------------------------------------------------------------------------+
| NVIDIA-SMI 535.104.05             Driver Version: 535.104.05   CUDA Version: 12.2     |
|-----------------------------------------+----------------------+----------------------+
| GPU  Name                 Persistence-M | Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp   Perf          Pwr:Usage/Cap |         Memory-Usage | GPU-Util  Compute M. |
|                                         |                      |               MIG M. |
|=========================================+======================+======================|
|   0  NVIDIA A100-SXM4-80GB          On  | 00000000:01:00.0 Off |                    0 |
| N/A   29C    P0              69W / 500W |      4MiB / 81920MiB |      0%      Default |
|                                         |                      |             Disabled |
+-----------------------------------------+----------------------+----------------------+
|   1  NVIDIA A100-SXM4-80GB          On  | 00000000:41:00.0 Off |                    0 |
| N/A   28C    P0              56W / 500W |      4MiB / 81920MiB |      0%      Default |
|                                         |                      |             Disabled |
+-----------------------------------------+----------------------+----------------------+
|   2  NVIDIA A100-SXM4-80GB          On  | 00000000:81:00.0 Off |                    0 |
| N/A   29C    P0              56W / 500W |      4MiB / 81920MiB |      0%      Default |
|                                         |                      |             Disabled |
+-----------------------------------------+----------------------+----------------------+
|   3  NVIDIA A100-SXM4-80GB          On  | 00000000:C1:00.0 Off |                   On |
| N/A   28C    P0              50W / 500W |     87MiB / 81920MiB |     N/A      Default |
|                                         |                      |              Enabled |
+-----------------------------------------+----------------------+----------------------+

+---------------------------------------------------------------------------------------+
| MIG devices:                                                                          |
+------------------+--------------------------------+-----------+-----------------------+
| GPU  GI  CI  MIG |                   Memory-Usage |        Vol|      Shared           |
|      ID  ID  Dev |                     BAR1-Usage | SM     Unc| CE ENC DEC OFA JPG    |
|                  |                                |        ECC|                       |
|==================+================================+===========+=======================|
|  3    7   0   0  |              12MiB /  9728MiB  | 14      0 |  1   0    0    0    0 |
|                  |               0MiB / 16383MiB  |           |                       |
+------------------+--------------------------------+-----------+-----------------------+
|  3    8   0   1  |              12MiB /  9728MiB  | 14      0 |  1   0    0    0    0 |
|                  |               0MiB / 16383MiB  |           |                       |
+------------------+--------------------------------+-----------+-----------------------+
|  3    9   0   2  |              12MiB /  9728MiB  | 14      0 |  1   0    0    0    0 |
|                  |               0MiB / 16383MiB  |           |                       |
+------------------+--------------------------------+-----------+-----------------------+
|  3   10   0   3  |              12MiB /  9728MiB  | 14      0 |  1   0    0    0    0 |
|                  |               0MiB / 16383MiB  |           |                       |
+------------------+--------------------------------+-----------+-----------------------+
|  3   11   0   4  |              12MiB /  9728MiB  | 14      0 |  1   0    0    0    0 |
|                  |               0MiB / 16383MiB  |           |                       |
+------------------+--------------------------------+-----------+-----------------------+
|  3   12   0   5  |              12MiB /  9728MiB  | 14      0 |  1   0    0    0    0 |
|                  |               0MiB / 16383MiB  |           |                       |
+------------------+--------------------------------+-----------+-----------------------+
|  3   13   0   6  |              12MiB /  9728MiB  | 14      0 |  1   0    0    0    0 |
|                  |               0MiB / 16383MiB  |           |                       |
+------------------+--------------------------------+-----------+-----------------------+
                                                                                         
+---------------------------------------------------------------------------------------+
| Processes:                                                                            |
|  GPU   GI   CI        PID   Type   Process name                            GPU Memory |
|        ID   ID                                                             Usage      |
|=======================================================================================|
|  No running processes found                                                           |
+---------------------------------------------------------------------------------------+
2024-10-21 10:25:38.214310: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcudart.so.11.0
<xarray.DataArray 'orog' ()>
array(1733.3604, dtype=float32) <xarray.DataArray 'he' ()>
array(1157.4626, dtype=float32) <xarray.DataArray 'vegt' ()>
array(17.000008, dtype=float32)
Traceback (most recent call last):
  File "/nesi/project/niwa00018/rampaln/envs/ml_env_v2/lib/python3.8/site-packages/xarray/backends/file_manager.py", line 199, in _acquire_with_cache_info
    file = self._cache[self._key]
  File "/nesi/project/niwa00018/rampaln/envs/ml_env_v2/lib/python3.8/site-packages/xarray/backends/lru_cache.py", line 53, in __getitem__
    value = self._cache[key]
KeyError: [<class 'netCDF4._netCDF4.Dataset'>, ('/scale_wlg_persistent/filesets/project/niwa00018/ML_downscaling_CCAM/On-the-Extrapolation-of-Generative-Adversarial-Networks-for-downscaling-precipitation-extremes/inputs/predictor/Other_GCMs_hist_SSP370_predictor_fields.nc',), 'r', (('clobber', True), ('diskless', False), ('format', 'NETCDF4'), ('persist', False))]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/nesi/project/niwa00018/ML_downscaling_CCAM/On-the-Extrapolation-of-Generative-Adversarial-Networks-for-downscaling-precipitation-extremes/model_inference/model_inference_all_gcms.py", line 80, in <module>
    stacked_X, y, vegt, orog, he = preprocess_input_data(config_test_data, match_index =False)
  File "/scale_wlg_persistent/filesets/project/niwa00018/ML_downscaling_CCAM/On-the-Extrapolation-of-Generative-Adversarial-Networks-for-downscaling-precipitation-extremes/src/process_input_training_data.py", line 55, in preprocess_input_data
    X = xr.open_dataset(config["train_x"])  # .sel(time = slice("2016", None))
  File "/nesi/project/niwa00018/rampaln/envs/ml_env_v2/lib/python3.8/site-packages/xarray/backends/api.py", line 497, in open_dataset
    backend_ds = backend.open_dataset(
  File "/nesi/project/niwa00018/rampaln/envs/ml_env_v2/lib/python3.8/site-packages/xarray/backends/netCDF4_.py", line 551, in open_dataset
    store = NetCDF4DataStore.open(
  File "/nesi/project/niwa00018/rampaln/envs/ml_env_v2/lib/python3.8/site-packages/xarray/backends/netCDF4_.py", line 380, in open
    return cls(manager, group=group, mode=mode, lock=lock, autoclose=autoclose)
  File "/nesi/project/niwa00018/rampaln/envs/ml_env_v2/lib/python3.8/site-packages/xarray/backends/netCDF4_.py", line 328, in __init__
    self.format = self.ds.data_model
  File "/nesi/project/niwa00018/rampaln/envs/ml_env_v2/lib/python3.8/site-packages/xarray/backends/netCDF4_.py", line 389, in ds
    return self._acquire()
  File "/nesi/project/niwa00018/rampaln/envs/ml_env_v2/lib/python3.8/site-packages/xarray/backends/netCDF4_.py", line 383, in _acquire
    with self._manager.acquire_context(needs_lock) as root:
  File "/nesi/project/niwa00018/rampaln/envs/ml_env_v2/lib/python3.8/contextlib.py", line 113, in __enter__
    return next(self.gen)
  File "/nesi/project/niwa00018/rampaln/envs/ml_env_v2/lib/python3.8/site-packages/xarray/backends/file_manager.py", line 187, in acquire_context
    file, cached = self._acquire_with_cache_info(needs_lock)
  File "/nesi/project/niwa00018/rampaln/envs/ml_env_v2/lib/python3.8/site-packages/xarray/backends/file_manager.py", line 205, in _acquire_with_cache_info
    file = self._opener(*self._args, **kwargs)
  File "src/netCDF4/_netCDF4.pyx", line 2470, in netCDF4._netCDF4.Dataset.__init__
  File "src/netCDF4/_netCDF4.pyx", line 2107, in netCDF4._netCDF4._ensure_nc_success
FileNotFoundError: [Errno 2] No such file or directory: '/scale_wlg_persistent/filesets/project/niwa00018/ML_downscaling_CCAM/On-the-Extrapolation-of-Generative-Adversarial-Networks-for-downscaling-precipitation-extremes/inputs/predictor/Other_GCMs_hist_SSP370_predictor_fields.nc'
